src/components/user-login/user-login.tsx

```diff
@@ -42,6 +54,21 @@ export const UserLogin = () => {
                     placeholder={avatar}
                     renderValue={() => avatar}
                 >
+                    <SelectItem
```

Comment:
using `Select` component for this purpose looks weird to me.. and quite confusing to read

Reply:
I wouldn't expect that editing `Select` component will result in change to the user menu in the header